examples/IRMQTTServer/IRMQTTServer.h

```diff
@@ -63,7 +63,7 @@
 #define BAUD_RATE 115200  // Serial port Baud rate.
 
 // Change if you need multiple independent send gpios & topics. (MQTT only)
-const uint8_t kNrOfIrTxGpios = 8;
+const uint8_t kNrOfIrTxGpios = 9;
```

Comment:
it is not mandatory, but it would awesome to replace this with ARRAY_OF, something like this:

```
#define ARRAY_SIZE(ARRAY) (sizeof(ARRAY)/sizeof((ARRAY)[0]))

#define kNrOfIrTxGpios ARRAY_SIZE(txGpioTable)
```

and send this patch upstream

Reply:
(I let a comment below)